### PR TITLE
Add a 'config_file' param with dynamic config change support

### PR DESCRIPTION
**Pull request by sbernard31**
It aims to implement #3.

The new param is :
```
  -cfg CONFIG_FILE, --config_file CONFIG_FILE
                        <Required> a path to a file containing real server address(es). 
                        File will be polled each second for modification and configuration
                        updated dynamically. A file content example :
                        
                        [Real Servers]
                        10.0.0.4
                        10.0.0.2
                        10.0.0.6
```

### CODE REVIEW

File: ulb.py [hunk 1]
```diff
@@ -62,13 +77,37 @@ def ip_parser(s):
...
+config_file_mtime = 0                # last modification time of config file
+
+def load_config(cfgFile):
+    """Load configuration from file object and return a list of server"""
+    print ("\nLoading real servers from {} file ...".format(cfgFile.name))
+    config = configparser.ConfigParser(allow_no_value=True)
+    config.read_file(cfgFile)
+    rs = []
+    for ip in config["Real Servers"]:
+        rs.append(ip_parser(ip))
```
Comment (oliwer) [inline ulb.py]: What happens if ip_parser() fails? Is the error ignored?

File: ulb.py [hunk 2]
```diff
@@ -97,31 +140,97 @@ class Data(ct.Structure):
...
+            # DEBUG STUFF
+            #(task, pid, cpu, flags, ts, msg) = b.trace_fields(nonblocking = True)
+            #while msg:
+            #    print("%s \n" % (msg))
+            #    (task, pid, cpu, flags, ts, msg) = b.trace_fields(nonblocking = True)
+        except KeyboardInterrupt:
+            break;
+finally:
+    # Detach bpf progam
+    b.remove_xdp(ifnet)
```
Comment (oliwer) [inline ulb.py]: Will the XDP also be removed if the process receives a SIGTERM? That's what systemd sends to stop a service.

Comment (sbernard31) [reply ulb.py]: Good question I can test this.

File: ulb.py (hunk 1 above)
Comment (sbernard31) [reply ulb.py]: exception is raised and will be catched by both `load_config` callers :+1: 
 - One at start-up and the script is stopped.
 - the other, when config change dynamically, in this case if we can loadconfig we just keep the previous one.

File: ulb.py (hunk 2 above)
Comment (sbernard31) [reply ulb.py]: Tested with `kill -SIGTERM PID` and looking at `ip link`
```
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 xdpgeneric qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    prog/xdp id 48 
```

XDP prog is still attached (=> prog/xdp id 48)

So I will create a PR for this.

Is SIGTERM the only one SIGNAL we want to handle ?

Comment (oliwer) [reply ulb.py]: Yes, SIGTERM and SIGINT are the most important to handle.

**Pull request merged**